development/images/bootstrap/Dockerfile

```
@@ -0,0 +1,22 @@
+FROM maven:3-eclipse-temurin-21 AS builder
+
+ARG JOB_VERSION
+RUN --mount=type=secret,id=mvnrepo,dst=./settings.xml mvn -s settings.xml dependency:copy -Dartifact=com.backbase.accelerators:bootstrap-job:${JOB_VERSION}:zip:src -DoutputDirectory=/tmp/
+
+RUN apt-get update && apt-get install -y unzip docker.io
```

```suggestion
RUN apt-get update && apt-get install -y unzip
```

Lets remove `docker.io`, it is unnecessary I left it there by mistake :)